webserver.py

```diff
@@ -146,7 +161,23 @@ def remove_device_name(name):
         """DELETE FROM names WHERE name=(?)""",
         [name]
     )
+    
+def add_note_to_db(note):
+    d = datetime.utcnow()
+    unixtime = calendar.timegm(d.utctimetuple())
```

Comment:
in the other code, I use `unixtime = int(time.time())` in place of these lines (if you decide to remove this, don't forget to remove the `import calendar` import)

Reply:
Changed and removed